# HG changeset patch
# User Rik <rik@octave.org>
# Date 1541829750 28800
#      Fri Nov 09 22:02:30 2018 -0800
# Node ID d02e4b1df697434a0f96ba2c24a295f06b7ab078
# Parent  1357b79f87bc09c85902469b8110435f7e39523d
doc: Spellcheck documentation.

* aspell-octave.en.pws: Add new words to Octave-specific dictionary.

* external.txi, preface.txi, open.m, fzero.m, print.m: Correct spellings
and add @nospell macro around words which shouldn't be checked.

diff --git a/doc/interpreter/doccheck/aspell-octave.en.pws b/doc/interpreter/doccheck/aspell-octave.en.pws
--- a/doc/interpreter/doccheck/aspell-octave.en.pws
+++ b/doc/interpreter/doccheck/aspell-octave.en.pws
@@ -1,13 +1,13 @@
 personal_ws-1.1 en 1
 AbsTol
 accumarray
+acknowledgements
 Acknowledgements
-acknowledgements
 adams
 Affero
 afterwards
 al
 aleph
 amd
 anisotropic
 API
@@ -46,18 +46,18 @@ AWK
 azimuthal
 backend
 backgroundcolor
 backtrace
 bartlett
 Barycentric
 basevalue
 BaseValue
+bdf
 BDF
-bdf
 benchmarking
 betacdf
 betainc
 betainv
 betaln
 betapdf
 betarnd
 BFGS
@@ -79,18 +79,18 @@ boxoff
 boxon
 bracketx
 brackety
 braindead
 breakpoint
 broadcastable
 BufSize
 builtin
+bulleted
 Bulleted
-bulleted
 ButtonDownFcn
 buttondownfcn
 BV
 BZ
 bzip
 CallbackObject
 callee's
 camd
@@ -138,49 +138,50 @@ codebases
 codepage
 coffset
 colamd
 CollectOutput
 colorbar
 colorcube
 colormap
 colormaps
+ColorOrder
 colororder
-ColorOrder
 colperm
+CommentStyle
 commentstyle
-CommentStyle
 ComplexEqn
 cond
 condest
 conformant
 cong
 const
 contextless
 contourc
 convhull
 Convolve
 copyrightable
+CorelDraw
 corrcoef
 cosecant
 courseware
 cov
 covariance
 createcancelbtn
 CreateFcn
 CRLF
 crout
 Crout
 cryptographic
 CSparse
 CSV
 csymamd
 ctranspose
+CTRL
 Ctrl
-CTRL
 cubehelix
 cummax
 cummin
 cumprod
 cumsum
 cURL
 CurrentAxes
 CurrentFigure
@@ -202,41 +203,42 @@ datestr
 datestrings
 ddd
 dddd
 deallocated
 deallocating
 deconv
 Deconvolve
 decrement
+defaultaxesbox
 defaultuicontrolbackgroundcolor
 deftypefn
+Delaunay
 delaunay
-Delaunay
 delaunayn
 DeleteFcn
 deletefcn
 delim
 deltaX
 demi
 det
 dggsvd
 diag
 diagcomp
+Dialogs
 dialogs
-Dialogs
 diamondsuit
 differentiable
 digamma
 dimensionality
 dimensionally
 dir
 disp
+displayname
 DisplayName
-displayname
 displayrange
 dMatrix
 dmperm
 dockcontrols
 docstring
 docstrings
 dOmega
 doNotSpecify
@@ -261,37 +263,37 @@ eigenvector
 eigenvectors
 eigs
 elementwise
 elseif
 emacs
 emptyvalue
 EmptyValue
 endfunction
+endian
 Endian
-endian
 endif
+EndOfLine
 endofline
-EndOfLine
 EOF
 EOLs
 eps
 eq
 equidistributed
+equilibration
 Equilibration
-equilibration
 equispaced
 erf
 erfc
 erfi
 errno
+errorbar
 Errorbar
-errorbar
+errorbars
 Errorbars
-errorbars
 errordlg
 ErrorHandler
 ESC
 et
 etree
 etreeplot
 eval
 evalc
@@ -302,58 +304,61 @@ expcdf
 expinv
 exppdf
 exprnd
 extendedtext
 extrema
 facecolor
 FaceColor
 FaceLighting
+FaceNormals
+FaceVertexCData
 facevertexcdata
-FaceVertexCData
 factorizations
 fcdf
 femmodel
 FFF
 FFT
 fft
 fftn
 fftpack
 FFTs
 fftw
 fieldname
 fieldnames
 FIFOs
 filename
+FileName
 filenames
 filepaths
 Filesystem
 filetype
 FinDiffType
 finv
 FIRfilter
 fitboxtotext
 FIXME
 flac
+FLTK
 fltk
-FLTK
+fminsearch
 fminunc
 fontconfig
 fontname
 forall
 foregroundcolor
 formfeed
 Fortran
 fpdf
 fprintf
 FreeBSD
 FreeSans
 freespacing
+FreeType
 freetype
-FreeType
 frnd
 Fs
 FSF
 fullpath
 fullpathext
 FunValCheck
 gamcdf
 gaminv
@@ -382,20 +387,20 @@ gif
 GIF
 glibc
 globbing
 glpk
 gls
 GLS
 glyphs
 GMRES
+gnuplot
 Gnuplot
-gnuplot
+Goto
 goto
-Goto
 gouraud
 gperf
 GPL
 GPLK
 gplot
 grabdemo
 GradObj
 GraphicsAlphaBits
@@ -403,40 +408,42 @@ GraphicsMagick
 grayscale
 griddata
 gswin
 gui
 GUIs
 gzip
 gzipped
 Hadamard
+handlevisibility
 HandleVisibility
-handlevisibility
 Hankel
 Hanning
 hanning
 hardcode
 hardcoded
 hardcoding
+hdf
 HDF
-hdf
+HeaderLines
 headerlines
-HeaderLines
 headlength
 headstyle
 headwidth
 heartsuit
 helpdlg
+Helvetica
 Hermitian
 Hessenberg
 heteroscedascity
 heteroscedastic
 hggroup
 hggroups
 hgid
+hgload
 HH
 hh
 histc
 holomorphic
 horizontalalignment
 horzcat
 hostname
 HSV
@@ -446,18 +453,18 @@ hygecdf
 hygeinv
 hygepdf
 hygernd
 hypervolume
 ichol
 ict
 IDE
 IEC
+IEEE
 ieee
-IEEE
 ifelse
 ifft
 ifftn
 ignorecase
 ij
 ilu
 ilutp
 Im
@@ -516,21 +523,21 @@ Jacobian
 Jacobians
 javaaddpath
 javamem
 Javascript
 jbig
 JConstant
 JDK
 JIS
+jit
 JIT
-jit
 JPattern
+jpeg
 JPEG
-jpeg
 jpg
 jvm
 JVM's
 kendall
 keybindings
 keypress
 keypressfcn
 KeyType
@@ -622,16 +629,17 @@ menubars
 MERCHANTABILITY
 Mersenne
 meshgrid
 meshgridded
 meshstyle
 metadata
 metafile
 Metafile
+MetaFile
 metafiles
 Metafont
 mex
 mget
 michol
 Microsystems
 MinGW
 minima
@@ -644,41 +652,42 @@ mkdir
 mkoctfile
 mldivide
 mmd
 mmm
 MMM
 mmmm
 mmmyy
 mmmyyyy
+movegui
 mpoles
 mpower
 mput
 mrdivide
 msgbox
 MStateDependence
 MSYS
 mtimes
 multi
 Multi
 multibyte
 multipage
+MultipleDelimsAsOne
 multipledelimsasone
-MultipleDelimsAsOne
 MultiSelect
 multistep
 MvPattern
 mxArray
 myclass
 myfun
 nabla
+NAMESPACE
 namespace
-NAMESPACE
+nan
 NaN
-nan
 nanflag
 NaNs
 nargin
 nargout
 nbincdf
 nbininv
 nbinpdf
 nbinrnd
@@ -702,16 +711,17 @@ nolabel
 noncommercially
 nonconformant
 nondecreasing
 nonincreasing
 nonmodal
 nonnan
 NonNegative
 nonnegativity
+nonskew
 nonsmooth
 nonsparse
 nonzeros
 NOP
 noperm
 norc
 normals
 normcdf
@@ -729,25 +739,28 @@ NTSC
 numbertitle
 numel
 Numpy
 nzmax
 oct
 octaverc
 ODEPACK
 ODEs
+offscreen
+ofig
+Ogg
 ogg
-Ogg
 OKString
 OLS
 omitnan
 onCleanup
 online
 OpenBLAS
 OpenGL
+opengl
 OpenJDK
 oplus
 optimizations
 Ord
 oregonator
 Orthogonalize
 orthonormal
 oslash
@@ -764,43 +777,43 @@ paperorientation
 paperposition
 PaperPosition
 paperpositionmode
 papersize
 paperunits
 PaperUnits
 parametrically
 parseparams
+pbm
 PBM
-pbm
 PBMplus
 pc
 pcg
 PCG
 pchip
 pcr
 PCRE
 pcre
 PCX
+pdf
 PDF
-pdf
 pdflatex
 pentadiagonal
 periodogram
 perp
 PGF
 PGMRES
 PHP
 pict
 piecewise
 Piecewise
 pinv
 PixelRegion
+PNG
 png
-PNG
 poisscdf
 poissinv
 poisspdf
 poissrnd
 polyderiv
 polyeig
 polyfit
 polygamma
@@ -821,18 +834,18 @@ Preformatted
 preformatted
 premultiplied
 Prepend
 prepended
 preselected
 presolver
 printf
 priori
+profiler
 Profiler
-profiler
 programmatically
 prolate
 PromptString
 propto
 ps
 pseudoinverse
 pseudonorm
 pseudorandom
@@ -840,87 +853,89 @@ Pseudospectra
 pseudospectrum
 ptrace
 pushbutton
 Pxx
 qhull
 Qhull
 QP
 QQ
+QRUPDATE
 qrupdate
-QRUPDATE
 QScintilla
 quadcc
 quadgk
 quadl
 quadpack
 quadv
+Quantile
 quantile
-Quantile
 quantiles
 quantization
 quartic
 quartile
 questdlg
 Quickhull
+qz
 QZ
-qz
 radian
 radians
 radices
 radiobutton
 radix
 rande
 randg
 randi
 randn
 randp
 randperm
 rangle
 ranlib
 rceil
 rdivide
+Readline
 readline
-Readline
 RECT
 recursing
 redistributable
 reentrant
 reflectance
 regex
 regressor
 relicensing
 RelTol
 renderer
+renderers
 repelems
 replacechildren
 repmat
 representable
 reproducibility
 resampled
 resampling
+resize
 Resize
-resize
 resized
 Resizing
 resizing
 resnorm
 resparsify
 restacks
 restoreBG
 restorePrevious
 RET
 retarget
+returnonerror
 ReturnOnError
-returnonerror
 rfloor
+rgb
 RGB
-rgb
+RGBImage
+rightarrow
 Rightarrow
-rightarrow
 rle
 rline
 rmdir
 RMS
 rnd
 roffset
 rpath
 RPMs
@@ -978,63 +993,64 @@ sprank
 SPSS
 spstats
 SQP
 sqrt
 sqrtm
 ss
 sT
 stairstep
+Startup
 startup
-Startup
 statinfo
 stdin
 stdout
 stepsize
 STFT
 strcmp
 stringanchors
 strncmp
 strncmpi
 strread
 strread's
 strsplit
 struct
 structs
 subarrays
+subclasses
 Subclasses
-subclasses
+subdiagonal
 subdiagonals
 subdirectories
 subdirectory
 subexpressions
 Subfunction
 subfunction
 Subfunctions
 subfunctions
 subinterval
+subintervals
 Subintervals
-subintervals
 sublicenses
 Sublicensing
 submatrices
 submatrix
 submenu
 suboptimal
 subprocess
+subprocesses
 Subprocesses
-subprocesses
 subsasgn
+subscripted
 Subscripted
-subscripted
 subscripting
 subseteq
 subsindex
+subspaces
 Subspaces
-subspaces
 subsref
 substring
 substrings
 subwindows
 SuiteSparse
 sumsq
 superclass
 Supercomputing
@@ -1042,25 +1058,25 @@ superdiagonal
 superdiagonals
 superiorto
 superscripted
 superscripting
 supset
 supseteq
 surd
 SV
+svd
 SVD
-svd
 svds
-svg
 SVG
-Sym
+svg
 sym
+Sym
+SYMAMD
 symamd
-SYMAMD
 symbfact
 symrcm
 Syntaxes
 tcdf
 terminal's
 tex
 texi
 texinfo
@@ -1087,18 +1103,18 @@ tinv
 tmp
 Toeplitz
 togglebutton
 togglebuttons
 tokenExtents
 TolF
 TolFun
 TolX
+Toolkits
 toolkits
-Toolkits
 toplevel
 TP
 tpdf
 traceback
 TRANSA
 trapz
 treatasempty
 TreatAsEmpty
@@ -1106,34 +1122,34 @@ treelayout
 treeplot
 tridiagonal
 trigamma
 trimesh
 triplot
 trisurf
 trivariate
 trnd
+TrueColor
 truecolor
-TrueColor
 tuples
 txi
 typedefs
 TypicalX
 ub
 UB
 uchar
 UHESS
 UI
+Uibuttongroup
 uibuttongroup
-Uibuttongroup
 uibuttongroups
 uicontextmenu
 Uicontextmenu
+Uicontrol
 uicontrol
-Uicontrol
 uicontrols
 UID
 uimenu
 uint
 uipanel
 Uipanel
 uipanels
 uipushtool
@@ -1143,18 +1159,18 @@ uiputfile
 Uitoggletool
 uitoggletool
 uitoolbar
 Uitoolbar
 ulong
 Ultrix
 umfpack
 uminus
+unary
 Unary
-unary
 unconvertible
 undirected
 unifcdf
 unifinv
 UniformOutput
 UniformValues
 unifpdf
 unifrnd
@@ -1207,55 +1223,57 @@ viewable
 viridis
 vorbis
 Vorbis
 voronoi
 Voronoi
 waitbar
 waitbars
 warndlg
+WAV
 wav
-WAV
-waypoints
 Waypoints
 WayPoints
+waypoints
 wblcdf
 wblinv
 wblpdf
 wblrnd
 WestOutside
+whitespace
 Whitespace
-whitespace
 whos
 wienrnd
 Wikipedia
 wildcard
+Wildcards
 wildcards
-Wildcards
 windowbuttondownfcn
 windowbuttonmotionfcn
 windowbuttonupfcn
+windowstyle
 WindowStyle
-windowstyle
 WIPO
 wireframe
 workspace
 wp
 xb
 xboxes
 xc
 xcorr
 xdata
 xerrorbar
 xerrorbars
 XFig
+xfirst
 xGBTRF
 xGELSD
 xGETRF
 xGTSV
+xlast
 xlim
 xlimmode
 xmax
 xmin
 xon
 xPBTRF
 xpos
 xPOTRF
@@ -1273,27 +1291,29 @@ xyboxes
 xyerrorbar
 xyerrorbars
 xyz
 yboxes
 yc
 ydata
 yerrorbar
 yerrorbars
+yfirst
 ylabel
+ylast
 ylim
 ymax
 ymin
 ypos
 yticklabels
 yticks
+YY
 yy
-YY
+yyyy
 YYYY
-yyyy
 yyyymmddTHHMMSS
 zggsvd
 Ziggurat
 zlabel
 zlabels
 zlib
 zlim
 zmax
diff --git a/doc/interpreter/external.txi b/doc/interpreter/external.txi
--- a/doc/interpreter/external.txi
+++ b/doc/interpreter/external.txi
@@ -2087,28 +2087,28 @@ javaObject ("java.math.BigDecimal", "1")
 @end group
 @end example
 
 
 
 @node Set up the JVM
 @subsection Set up the JVM
 @cindex memory, limitations on JVM
-@cindex select JVM vesion
+@cindex select JVM version
 
 In order to execute Java code Octave creates a Java Virtual Machine (JVM).  By
 default the version of the JVM is used that was detected during configuration
 on Unix-like systems or that is pointed to from the registry keys at
 @file{HKEY_LOCAL_MACHINE\SOFTWARE\JavaSoft\JRE} or
 @file{HKEY_LOCAL_MACHINE\SOFTWARE\JavaSoft\Java Runtime Environment} on
 Windows.  The default path to the JVM can be overwritten by setting the
 environment variable @w{@env{JAVA_HOME}} to the path where the JVM is
 installed.  On Windows that might be, for example,
 @file{C:\Program Files\Java\jre-10.0.2}.  Make sure that you select a directory
-that contains the JVM with a bit-ness that matches Octave's.
+that contains the JVM with a @nospell{bit-ness} that matches Octave's.
 
 The JVM is only loaded once per Octave session.  Thus, to change the used
 version of the JVM, you might have to re-start Octave.  To check which version
 of the JVM is currently being used, run @code{version -java}.
 
 The JVM allocates a fixed amount of initial memory and may expand this pool up
 to a fixed maximum memory limit.  The default values depend on the Java version
 (@pxref{XREFjavamem,,javamem}).  The memory pool is shared by all Java objects
diff --git a/doc/interpreter/preface.txi b/doc/interpreter/preface.txi
--- a/doc/interpreter/preface.txi
+++ b/doc/interpreter/preface.txi
@@ -94,17 +94,17 @@ The United States Department of Energy, 
 @nospell{Stanley C. Ahalt} of the Ohio Supercomputer Center.
 
 @item
 The National Science Foundation, through grant numbers @nospell{CTS-0105360},
 @nospell{CTS-9708497, CTS-9311420, CTS-8957123, and CNS-0540147}.
 
 @item
 The industrial members of the Texas-Wisconsin Modeling and Control
-Consortium (TWMCC).
+Consortium (@nospell{TWMCC}).
 
 @item
 The @nospell{Paul A. Elfers} Endowed Chair in Chemical Engineering at the
 University of Wisconsin-Madison.
 
 @item
 Digital Equipment Corporation, for an equipment grant as part of their
 External Research Program.
diff --git a/scripts/miscellaneous/open.m b/scripts/miscellaneous/open.m
--- a/scripts/miscellaneous/open.m
+++ b/scripts/miscellaneous/open.m
@@ -44,19 +44,19 @@
 ## @item .exe
 ## Execute the program (on Windows systems only).  No @var{output} value
 ## is returned.
 ## @end table
 ##
 ## Custom file extensions may also be handled if a function @code{openxxx},
 ## where @code{xxx} is the extension, is found in the load path.  The function
 ## must accept the file name as input.  For example, in order to load
-## @qcode{".dat"} data files in the base workspace, as is done by default
-## for @qcode{".mat"} files, one may define @qcode{"opendat.m"} with the
-## following contents:
+## @nospell{@qcode{".dat"}} data files in the base workspace, as is done by
+## default for @qcode{".mat"} files, one may define
+## @nospell{@qcode{"opendat.m"}} with the following contents:
 ##
 ## @example
 ## @group
 ## function retval = opendat (fname)
 ##   evalin ("base", sprintf ("load ('%s');", fname));
 ## endfunction
 ## @end group
 ## @end example
diff --git a/scripts/optimization/fzero.m b/scripts/optimization/fzero.m
--- a/scripts/optimization/fzero.m
+++ b/scripts/optimization/fzero.m
@@ -82,17 +82,17 @@
 ##
 ## @var{output} is a structure containing runtime information about the
 ## @code{fzero} algorithm.  Fields in the structure are:
 ##
 ## @itemize
 ## @item iterations
 ##  Number of iterations through loop.
 ##
-## @item funcCount
+## @item @nospell{funcCount}
 ##  Number of function evaluations.
 ##
 ## @item algorithm
 ##  The string @qcode{"bisection, interpolation"}.
 ##
 ## @item bracketx
 ##  A two-element vector with the final bracketing of the zero along the
 ## x-axis.
diff --git a/scripts/plot/util/print.m b/scripts/plot/util/print.m
--- a/scripts/plot/util/print.m
+++ b/scripts/plot/util/print.m
@@ -303,17 +303,18 @@
 ## the @option{-svgconvert} option.
 ##
 ##   @item svg
 ##     Scalable Vector Graphics
 ##
 ##   @item  tif
 ##   @itemx tiff
 ##   @itemx tiffn
-##     TIFF image with LZW compression (tif, tiff) or uncompressed (tiffn).
+##     TIFF image with LZW compression (@nospell{tif}, tiff) or uncompressed
+## (@nospell{tiffn}).
 ##
 ##   @item  tikz
 ##   @itemx tikzstandalone*
 ##     Generate a @LaTeX{} file using PGF/TikZ format.  The OpenGL-based
 ## toolkits create a PGF file while Gnuplot creates a TikZ file.  The
 ## @samp{tikzstandalone} device produces a @LaTeX{} document which includes the
 ## TikZ file.
 ##   @end table
